src/containers/audio-rewards-page/components/ButtonWithArrow.module.css

```diff
@@ -0,0 +1,7 @@
+.rewardButton {
+  padding: 8px 12px;
+  height: 34px !important;
+}
+.buttonIcon {
+  margin: 0 2px 0 0px !important;
```

Comment:
Would you still need `!important` if this was `.rewardButton .buttonIcon {` 
I'm not sure, but I don't like important, but also I use it it a lot 🙃 . 
Feel free to leave as it.

Reply:
will test

Reply:
no dice :(